app/views/bulkrax/shared/_entries_tab.html.erb

```diff
@@ -12,5 +12,6 @@
       </tr>
     </thead>
   </table>
-  <div id='importer-entry-classes' class='hidden'><%= [@importer.parser.entry_class.to_s, @importer.parser.collection_entry_class.to_s, @importer.parser.file_set_entry_class.to_s].compact.join('|') %></div>
+  <% agent = instance_variable_get("@#{controller.controller_name.singularize}") %>
```

Comment:
this is clever, but it is also likely to be hard to debug. instead pass the importer or exporter in from the show page to a variable used in the partial